openpgp/internal/algorithm/hash.go

```diff
@@ -8,6 +8,8 @@ import (
 	"crypto"
 	"fmt"
 	"hash"
+
+	_ "crypto/md5"
```

Comment:
Why this ignored import?

Reply:
This is only to link `crypto.MD5` to the binary (it was verbatim in some `_v3.go` file). Without this import, a user using `crypto` gets an error when calling `NewEntity` with MD5 as a default hash (and the current `config` API). We should remove MD5 altogether, but this is only to stick with current behavior for now.

Reply:
Great, sounds reasonable to me :)

Reply:
Yep, I don't quite like this, but a future PR will just remove MD5 and this import